wrapanapi/containers/providers/rhopenshift.py

```diff
@@ -23,6 +24,34 @@
 from wrapanapi.containers.deployment_config import DeploymentConfig
 
 
+# stolen from sprout
+VERSION_REGEXPS = [
+    r"^cfme-(\d)(\d)(\d)(\d)(\d{2})",  # 1.2.3.4.11
```

Comment:
**SUGESTION:**  Is there anyway we could encapsulate these in a library that we use in both places.   I see great potential for changing one of these in one, but forgetting to do so here.

Reply:
cool...and thanks!